src/container/srv_target.c

```diff
@@ -86,6 +86,7 @@ cont_child_alloc_ref(void *key, unsigned int ksize, void *varg,
 		goto out_cond;
 
 	uuid_copy(cont->sc_uuid, key);
+	cont->sc_pool = pool;
```

Comment:
Hold a ref?

Reply:
oh, container/pool cache management cleanup needs to be done in a different patch.